src/apps/loyalty/server/middlewares/session.ts

```diff
@@ -0,0 +1,11 @@
+import { NextFunction, Request, Response } from "express"
```

Comment:
This is more about 'bootstrapping to sharify' rather than really doing anything session related, so I can probably rename.